src/index.js

```diff
@@ -73,10 +83,13 @@ export default function() {
         let foundNegativeDirective = false;
         path.traverse({
           DirectiveLiteral(path) {
-            if (path.node.value === 'use superstrict') {
-              foundPositiveDirective = true;
-            } else if (path.node.value === 'use !superstrict') {
-              foundNegativeDirective = true;
+            const directives = splitMultipleDirectives(path.node.value);
```

Comment:
Why is this within path.traverse? Is it allowed to put such directives elsewhere than the Program node? Please copy the directive parsing from extensible-destructuring plugin (or create a PR, if you don't find that approach good).